djstripe/stripe_objects.py

```diff
@@ -89,7 +89,7 @@ def api_retrieve(self, api_key=settings.STRIPE_SECRET_KEY):
         """
 
         # Run stripe.X.retreive(id)
-        return type(self)._api().retrieve(id=self.stripe_id, api_key=api_key, expand=self.expand_fields)
+        return type(self)._api().retrieve(self.stripe_id, api_key=api_key, expand=self.expand_fields)
```

Comment:
Seems to be OK - Must have been the failure on the mock asserts I was thinking about.